foxpuppet/tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from foxpuppet.foxpuppet import FoxPuppet
+from foxpuppet.ui.browser.window import Browser_Window
```

Comment:
Do we need the 'ui' part in the module hierarchy? I thought that foxpuppet should only care about ui.